ovirt/resource_ovirt_vm.go

```diff
@@ -517,6 +518,47 @@ func resourceOvirtVMUpdate(d *schema.ResourceData, meta interface{}) error {
                 return err_updateresult
         }
 
+	// Check status and Start/Stop VM
+	status, statusOK := d.GetOk("status")
+
+	if d.HasChange("status") && statusOK {
+	        // Try to start VM
+	        log.Printf("[DEBUG] Try to update runing status for VM (%s)", d.Id())
+		var vm_status ovirtsdk4.VmStatus
+
+		if status == "up" {
+			vm_status = ovirtsdk4.VMSTATUS_UP
+	                _, err = vmService.Start().Send()
+                	if err != nil {
+        	                return err
+	                }
+		} else {
+			vm_status = ovirtsdk4.VMSTATUS_DOWN
+                        _, err = vmService.Stop().Send()
+                        if err != nil {
+                                return err
+                        }
+		}
+
+	        // Wait until vm is update status
+	        log.Printf("[DEBUG] Wait for VM (%s) status to become %s", d.Id(), vm_status)
+
+	        upStateConf := &resource.StateChangeConf{
```

Comment:
`upStateConf` should be `desiredStateConf` , because it's not necessarily going up .

Reply:
Changed variable name to the desiredStateConf.
Commit https://github.com/oVirt/terraform-provider-ovirt/pull/198/commits/7fa45520ca4c370cc765d9bd9a0883fea7e409ce